bugs/sru-19.4.33-manual-4.txt

```diff
@@ -0,0 +1,306 @@
+=== Begin SRU Template ===
+[Impact]
+Images which provide multiple ssh authorized key files for users via
+setting AuthorizedKeysFiles = <file1> <file2> would result in an invalid ssh
+configuration resulting in no remote ssh connectivity due to the absence of
+any .ssh/authorized_keys file.
+
+In this scenario, old/broken cloud-init would emit
```

Comment:
"subdirectory" is in this sentence twice, which suggests to me it's mid-edit. Can we clarify this wording, please?